docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-mqtt.md

```diff
@@ -0,0 +1,135 @@
+| receiver          | string                    | Name of the contact point                                                       |
+| status            | string                    | Current status of the alert, `firing` or `resolved`                             |
+| orgId             | number                    | ID of the organization related to the payload                                   |
+| alerts            | array of [alerts](#alert) | Alerts that are triggering                                                      |
+| groupLabels       | object                    | Labels that are used for grouping, map of string keys to string values          |
+| commonLabels      | object                    | Labels that all alarms have in common, map of string keys to string values      |
+| commonAnnotations | object                    | Annotations that all alarms have in common, map of string keys to string values |
+| externalURL       | string                    | External URL to the Grafana instance sending this webhook                       |
+| version           | string                    | Version of the payload                                                          |
+| groupKey          | string                    | Key that is used for grouping                                                   |
+| message           | string                    | Rendered message of the alerts                                                  |
+
+### Alert
+
+Each alert in the `alerts` array has the following fields:
+
+| Key          | Type   | Description                                                                        |
+| ------------ | ------ | ---------------------------------------------------------------------------------- |
+| status       | string | Current status of the alert, `firing` or `resolved`                                |
+| labels       | object | Labels that are part of this alert, map of string keys to string values            |
+| annotations  | object | Annotations that are part of this alert, map of string keys to string values       |
+| startsAt     | string | Start time of the alert                                                            |
+| endsAt       | string | End time of the alert, default value when not resolved is `0001-01-01T00:00:00Z`   |
+| values       | object | Values that triggered the current status                                           |
+| generatorURL | string | URL of the alert rule in the Grafana UI                                            |
+| fingerprint  | string | The labels fingerprint, alarms with the same labels will have the same fingerprint |
+| silenceURL   | string | URL to silence the alert rule in the Grafana UI                                    |
+| dashboardURL | string | **Will be deprecated soon**                                                        |
```

Comment:
Do we know when? If not - "Deprecated. It will be removed in a future release."

Reply:
Changed to "Deprecated. It will be removed in a future release."